_source/_posts/2022-01-20-vue-applications-nuxt.md

```diff
@@ -0,0 +1,355 @@
+---
+layout: blog_post
+title: "Simplify Building Vue Applications with NuxtJS"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A tutorial that shows how to create a web application with Nuxt and Okta."
+tags: [vue, nuxt, javascript]
+tweets:
+- "Curious about @nuxt_js? Learn how to create a secure @vuejs app using @okta and Nuxt! 🚀"
+image: blog/vue-applications-nuxt/vue-nuxt-social.png
+type: conversion
+github: https://github.com/oktadev/okta-vue-nuxt-example
+---
+
+[Nuxt](https://nuxtjs.org/) calls itself *The intuitive Vue framework*. It aims to make a developer-friendly experience while not sacrificing performance or degrading the integrity of your architecture. It has been exciting to see the community and tooling around VueJS grow and evolve — there's no better time to get started in this ecosystem than now.   
+
+In this tutorial, you will build a small web application that retrieves some posts from an API and displays them for authenticated users. For authentication, you will integrate Okta into your Nuxt application. Okta's simple authentication system and the power of Nuxt means you can configure and set up your authentication in just a few moments.  
+
+{% img blog/vue-applications-nuxt/vue-nuxt-social.png alt:"Cover image with a screenshot of the finished Nuxt app." width:"800" %}{: .center-image }
+
+**What you'll need**
+
+- Your favorite IDE (I will be using [Visual Studio Code](https://code.visualstudio.com/))
+- [Node.js](https://nodejs.org/en/download/)
+- An [Okta Developer Account](https://developer.okta.com/signup/)
+- The [Okta CLI](https://cli.okta.com/) tool
+
+## Setup your Okta Application
+
+{% include setup/cli.md type="spa" framework="Vue"
+   loginRedirectUri="http://localhost:3000/login" logoutRedirectUri="http://localhost:3000/" signup="false" %}
+
+## Build your web application with Nuxt
+
+If you wish to follow along using the completed project you can [check out the git repository here](https://github.com/oktadev/okta-vue-nuxt-example).  
```

Comment:
The README links to "Create an Angular app" when it should probably say "Create a Vue app" and link to https://developer.okta.com/docs/guides/sign-into-spa/vue/main/#create-an-okta-app-integration instead.